src/Symfony/Bridge/Twig/Mime/TemplatedEmail.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Mime;
+
+use Symfony\Component\Mime\Email;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 4.3
+ */
+class TemplatedEmail extends Email
+{
+    private $template;
+    private $htmlTemplate;
+    private $textTemplate;
+    private $context = [];
+
+    public function template(?string $template): self
```

Comment:
Returning `self` is not accurate for fluent interfaces and will make IDE lose the chain when using child classes.
We're using no-return-type + `@return $this` in the code base in this situation currently.

Reply:
fixed